graylog2-web-interface/src/components/common/HoverForHelp.tsx

```diff
@@ -86,6 +87,7 @@ HoverForHelp.propTypes = {
   pullRight: PropTypes.bool,
   title: PropTypes.string,
   testId: PropTypes.string,
+  trigger: PropTypes.arrayOf(PropTypes.string),
```

Comment:
As far as I understand OverlayTrigger>['trigger'] might be just one single `string`